.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We generally don't vendor settings from various IDEs.

Reply:
Ack, we have devcontainer stuff in Envoy, so wasn't sure if it was acceptable here.